src/plugins/intel_cpu/src/nodes/multinomial.cpp

```diff
@@ -0,0 +1,176 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "multinomial.hpp"
+
+#include "ie_ngraph_utils.hpp"
+#include "openvino/op/multinomial.hpp"
+
+namespace ov {
+namespace intel_cpu {
+namespace node {
+
+Multinomial::Multinomial(const std::shared_ptr<ov::Node>& op, const GraphContext::CPtr& context)
```

Comment:
It looks like, this node should be executed even if all the inputs are constant. So need to set `constant = ConstantType::NoConst;`

Reply:
Right, added!